spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -259,6 +259,12 @@
 
           before do
             enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
+            # Below sets in_person_verification_pending_at
```

Comment:
Should this be done in [the factory trait for pending in-person enrollments](https://github.com/18F/identity-idp/blob/e92a99f5f3c93c011ef4e8a8d4e0b98be1435a6e/spec/factories/in_person_enrollments.rb#L14-L19)?

Reply:
We have a ticket for refactoring the `in_person_enrollments` factory, [LG-14237](https://cm-jira.usa.gov/browse/LG-14237).  I agree that this isn't the ideal set-up, but due to the time-sensitive nature of the bug, this was the approach we took.